src/components/10-atoms/link/index.js

```diff
@@ -80,7 +80,7 @@ class AXALink extends LitElement {
           if (typeof this.onClick === 'function') {
             this.onClick();
           } else {
-            window.location.href = this.href;
+            window.open(this.href, this.external ? '_blank' : '_top');
           }
         }}"
         target="${this.external ? '_blank' : '_top'}"
```

Comment:
This line seems to be obsolet now. I think every browser can handle window.open and target is not a "must" for a tag.

Reply:
Right, but I left it there to make the intent of the link obvious. Maybe this helps a screenreader or something.

Reply:
I would leave it in case we refactor the JS or in case the JS is not working, better to have it as HTML as well